data/events.yaml

```diff
@@ -39,3 +39,75 @@ x-events:
   properties:
     cluster_id: UUID
     host_id: UUID
+
+- name: cluster_registration_failed
+  message: "Failed to register cluster with name '{cluster_name}'. Error: {error}"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+    error: string
+
+- name: registered_cluster
+  message: "Registered cluster {cluster_name}"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+
+- name: add_hosts_cluster_registration_failed
+  message: "Failed to register add-hosts cluster with name '{cluster_name}'. Error: {error}"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+    cluster_name: string
+    error: string
+
+- name: registered_add_hosts_cluster
+  message: "Registered add-hosts cluster {cluster_name}"
```

Comment:
Merge this `registered_add_hosts_cluster` with `registered_cluster` event and add the ClusterKind property

Reply:
IMO the user doesn't have any benefit from knowing that the cluster is "add-hosts" cluster (let's say he actually understand that, although I believe he won't).
So I think the event could be simpler with a single "Register cluster" as the primer event. The property of kind could be added for our usage.